website/static/js/markdown.js

```diff
@@ -31,11 +32,33 @@ var bootstrapTable = function(md) {
 var oldMarkdownList = function(md) {
     md.block.ruler.after('hr', 'pyMarkdownList', pymarkdownList);
 };
+
+
+var ViewOnlyImage = function(md) {
+    var defaultRenderer = md.renderer.rules.image;
+    md.renderer.rules.image = function (tokens, idx, options, env, self) {
+        var token = tokens[idx];
+        var imageLink = token.attrs[token.attrIndex('src')][1];
+        var WaterbuterRegex = new RegExp(window.contextVars.waterbutlerURL + 'v1\/resources\/[a-zA-Z0-9]{1,}\/providers\/[a-z0-9]{1,}\/');
```

Comment:
Drive-by: This regex should only need to be instantiated once. Also, it should probably be a constant (and therefore use `UPPER_CASE`).